site/src/docs/components/accordion/index.mdx

```diff
@@ -25,6 +25,8 @@ export default ({ children, pageContext }) => <TabsLayout pageContext={pageConte
 
 - Use accordions to allow the user to quickly glance at the information and then choose to open sections that are interesting to them.
 - **Accordions must be given a header that describes the accordion content.** This header also acts as the label for the expand button. The header level can be chosen depending on the structure of the page.
+  - The `headingLevel` given to the Accordion should be higher than the header in the page content preceeding it (if there is one). Making the content semantically correct. For example if the Accordion is nested inside content with a header `h2` then the `headingLevel` for the Accordion should be 3.
```

Comment:
This sentence seems a bit out of place "Making the content semantically correct.", I assume it's referring to the previous sentence? 🤔

Reply:
Yeah, maybe I'll rephrase this a bit :D